bin/autojump

```diff
@@ -135,10 +135,13 @@ def add_path(data, path, weight=10):
     with resulting duplicate entries in the database than a single canonical
     path.
     """
-    path = unico(path).rstrip(os.sep)
+    path = os.path.abspath(path)
```

Comment:
Ahh nm, when I spot checked this in the console I goofed. Good call on the `os.path.abspath`, but we still need the end result to be encoded in unicode.